pkgs/applications/editors/jupyter-kernels/iruby/update.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env sh
+
+# First, manually update Gemfile
+
+# Regenerate Gemfile.lock
+nix shell .#bundler -c bundle lock
```

Comment:
Can we do this if we have not enabled flake?

Reply:
Probably not, but since this is a script for maintainers I think it's probably fine.